static_src/components/upsi/upsi_panel.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+
+import { appPropType } from '../../stores/app_store';
+import { upsisRequestPropType } from '../../stores/upsi_store';
+import Loading from '../loading.jsx';
+import ErrorMessage from '../error_message.jsx';
+import Panel from '../panel.jsx';
+import ComplexList from '../complex_list.jsx';
+import ElasticLine from '../elastic_line.jsx';
+import ElasticLineItem from '../elastic_line_item.jsx';
+
+const propTypes = {
+  app: appPropType.isRequired,
+  upsisRequest: upsisRequestPropType.isRequired
+};
+
+export default class UPSIPanel extends Component {
+  renderBound() {
+    const { app, upsisRequest } = this.props;
+    const { items: upsis } = upsisRequest;
+
+    const serviceGuids = new Set(app.services.map(s => s.guid));
+    const bound = upsis.filter(({ guid }) => serviceGuids.has(guid));
+
+    return (
+      <ComplexList
+        title="Bound to app"
+        emptyMessage={!bound.length ? <div>No instances</div> : null}
+      >
+        {bound.map(({ guid, name }) => (
+          <ElasticLine key={guid}>
+            <ElasticLineItem title={`ID: ${guid}`}>{name}</ElasticLineItem>
+            <ElasticLineItem align="end" />
+          </ElasticLine>
+        ))}
+      </ComplexList>
+    );
+  }
+
+  renderOther() {
```

Comment:
renderOther is kind of generic.
change renderOther to something like renderUnbound

Reply:
How about `renderOthersInSpace`? It's not quite "unbound" per:

> It's worth noting that the second section is not "unbound UPSI" - it's showing all UPSI in the space that are not bound to the app. In fact, it might be showing UPSI that are bound to another app. It would be nice to make this a bit better, but we're limited by what the CF API returns (it doesn't tell us if the UPSI is bound). So without doing some transformations ourselves, I found the best way to do this quickly was to just label the second section as "Others in the space".